client/src/modules/purchases/modals/search.tmpl.html

```diff
@@ -1,88 +1,111 @@
-          <strong translate>FORM.LABELS.STATUS</strong>
-        </p>
-
-        <label class="checkbox-inline">
-          <input type="checkbox" name="is_confirmed" ng-true-value="1" ng-false-value="0" ng-model="$ctrl.bundle.is_confirmed">
-          <span translate>PURCHASES.STATUS.CONFIRMED</span>
-        </label>
-
-        <label class="checkbox-inline">
-          <input type="checkbox" name="is_received" ng-true-value="1" ng-false-value="0" ng-model="$ctrl.bundle.is_received">
-          <span translate>PURCHASES.STATUS.RECEIVED</span>
-        </label>
-
-        <label class="checkbox-inline">
-          <input type="checkbox" name="is_cancelled" ng-true-value="1" ng-false-value="0" ng-model="$ctrl.bundle.is_cancelled">
-          <span translate>PURCHASES.STATUS.CANCELLED</span>
-        </label>
-      </div>
-    </fieldset>
+  <div class="modal-body search-modal">
+    <uib-tabset>
+      <uib-tab index="0" heading="{{ 'FORM.LABELS.SEARCH_QUERRIES' | translate}}" data-custom-filter-tab>
+        <div class="tab-body">
+          <div class="form-group" ng-class="{ 'has-error' : ModalForm.$submitted && ModalForm.reference.$invalid }">
+            <label class="control-label" translate> FORM.LABELS.REFERENCE </label>
+            <bh-clear on-clear="ModalCtrl.clear('reference')"></bh-clear>
+
+            <input type="text" class="form-control" name="reference" ng-model="ModalCtrl.params.reference">
+            <div class="help-block" ng-messages="ModalForm.reference.$error" ng-show="ModalForm.$submitted">
+              <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+            </div>
+          </div>
+
+          <!-- supplier -->
+          <div class="form-group">
+            <label class="control-label" translate>FORM.LABELS.SUPPLIER</label>
+            
+            <bh-clear on-clear="ModalCtrl.clear('supplier_uuid')"></bh-clear>
+
+            <ui-select name="supplier" ng-model="ModalCtrl.params.supplier_uuid">
+              <ui-select-match>
+                <span>{{$select.selected.display_name}}</span>
+              </ui-select-match>
+              <ui-select-choices ui-select-focus-patch repeat="supplier.uuid as supplier in ModalCtrl.suppliers | filter:$select.search">
+                <span ng-bind-html="supplier.display_name | highlight:$select.search"></span>
+              </ui-select-choices>
+            </ui-select>
+          </div>
+
+          <!-- user -->
+          <bh-user-select
+            user-id="ModalCtrl.bundle.user_id"
+            name="user_id"
+            on-select-callback="ModalCtrl.onSelectUser(user)">
+            <bh-clear on-clear="ModalCtrl.clear('user_id')"></bh-clear>
+          </bh-user-select> 
+
+
+          <!-- status -->
+          <div class="form-group">
+            <p class="control-label" style="margin-bottom:5px;">
+              <strong translate>FORM.LABELS.STATUS</strong>
+            </p>
+
+            <label class="checkbox-inline">
```

Comment:
Are you sure these should be checkboxes?  It looks like they do not work as expected.

For example, in the test dataset, search of "confirmed" yields 1 result.  Searching for both "confirmed" and "Received In Stock" should show _at least_ one, but nothing matches.
![confirmedandreceivedinstock](https://user-images.githubusercontent.com/896472/28961066-d9c96d8a-78f8-11e7-94fd-6e4721d6543d.png)
_Fig 1: Confirmed and Received Selection_